src/amr/data/field/coarsening/field_coarsen_operator.hpp

```diff
@@ -152,9 +152,11 @@ namespace amr
             if constexpr (dimension == 1)
             {
                 for (int ix = startIndex[dirX]; ix <= endIndex[dirX]; ++ix)
-                {
-                    coarsener(sourceField, destinationField, {{ix}});
-                }
+
+                    for (int ix = startIndex[dirX]; ix <= endIndex[dirX]; ++ix)
```

Comment:
suspect

Reply:
right.... copy paste that ended up there for no good reason